spec/IndexFolder_Oncocore/index.html

```diff
@@ -8,48 +8,210 @@ <h1>Contents of this Implementation Guide</h1>
 
-<p>In late 2018, ASCO convened committee of twenty leading clinical experts in oncology, radiology, surgery, and public health developed two use cases which drove the initial clinical data requirements for mCODE.
+<p>In late 2018, ASCO convened committee of twenty leading clinical experts in oncology, radiology, surgery, and public health developed two use cases that drove the initial clinical data requirements for mCODE.
 <ul>
     <li><b>Use Case 1</b>: <a href="mCODE_UseCase_RCC.docx">Comparative Effectiveness Analysis and Cooperative Decision Making</a></li>
     <li><b>Use Case 2</b>: <a href="mCODE_UseCase_NGS.docx">CEA with Next Generation Sequencing (NGS)</a></li>
-<!-- Need more information about the use cases -->
 </ul>
-<p>In early 2019, an open survey was conducted to validate and prioritize the data elements from these use cases. Further down-scoping was done based on whether the data would be stored or capture in an electronic health record (EHR), or if it placed undue documentation burden on the clinician.</p> 
+<p>While mCODE ultimately is meant to be applicable to across all types of cancer, the initial focus (and both use cases) has been on solid tumors.</p>
+</p>
+<p>In addition to information obtained from subject matter experts, several pre-existing standards, nomenclatures, and guidelines were consulted in the development of this specification, including:
+    <ul>
+        <li><a href="https://cancerstaging.org/references-tools/deskreferences/Pages/Breast-Cancer-Staging.aspx" target="_blank">American Joint Committee on Cancer (AJCC) Staging Manual (8th Edition) Breast Cancer Chapter</a></li>
+        <li><a href="https://www.cap.org/protocols-and-guidelines" target="_blank">College of American Pathologists (CAP) Cancer Protocols</a></li>
+        <li><a href="https://www.naaccr.org/SSDI/SSDI-Manual.pdf?v=1531675132" target="_blank">North American Association of Central Cancer Registries (NAACCR) 2018 Site-Specific Data Items Manual</a></li>
+        <li><a href="http://www.hl7.org/implement/standards/product_brief.cfm?product_id=383" target="_blank">HL7 CDA R2 IG: Reporting to Public Health Cancer Registries from Ambulatory Healthcare Providers</a></li>
+        <li><a href="https://www.nccn.org/professionals/physician_gls/default.aspx#site" target="_blank">National Comprehensive Cancer Network (NCCN) Clinical Practice Guidelines in Oncology</a></li>
+        <li><a href="http://recist.eortc.org/wp-content/uploads/2015/03/RECISTGuidelines.pdf" target="_blank">RECIST Guidelines v1.1</a></li>
```

Comment:
URL is broken.  Replace with https://project.eortc.org/recist/wp-content/uploads/sites/4/2015/03/RECISTGuidelines.pdf.

Reply:
fixed.